cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -94,17 +94,20 @@ class wrap_ugettext_with_xyz(object):  # pylint: disable=invalid-name
 
             def __init__(self, module):
                 self.module = module
-                self.old_ugettext = module.ugettext
+                # figuring out gettext variant to support py2 and py3
+                self.gettext_variant = 'ugettext' if six.PY2 else 'gettext'
```

Comment:
You don't need to save the variant name as a part of the object.  You can just have a local variable when determining this.

Reply:
Thanks for the review! I changed the code as per your suggestion.